packages/website/components/SidebarLink.tsx

```diff
@@ -10,13 +10,13 @@ export const getSectionTitleStyles = (isActive = false, indent = 1) => {
     sidebarItem: css({
       padding: `${tokens.spacingXs} ${tokens.spacingM}`,
       paddingLeft: `calc(${indent} * ${tokens.spacingM})`,
-      fontSize: tokens.fontSizeL,
-      lineHeight: tokens.lineHeightL,
+      fontSize: tokens.fontSizeM,
+      lineHeight: tokens.lineHeightM,
     }),
     link: css({
       display: 'block',
-      fontSize: `${tokens.fontSizeL}`,
-      lineHeight: `${tokens.lineHeightL}`,
+      fontSize: `${tokens.fontSizeM}`,
+      lineHeight: `${tokens.lineHeightM}`,
```

Comment:
Isn't it possible to use `<Text/>` for this?

Reply:
good point. I will try to do that. Thx!

Reply:
Let's follow up in the next PR. it will be subheading finally, so maybe let's do it in the next PR as soon as we have font sizes not overwritten.